sys/net/sock/sock_util.c

```diff
@@ -116,30 +117,33 @@ static char* _find_pathstart(const char *url)
 
 int sock_urlsplit(const char *url, char *hostport, char *urlpath)
 {
+    assert(url != NULL);
     char *hoststart = _find_hoststart(url);
     if (!hoststart) {
```

Comment:
as you changed the others this should be `if (hoststart == NULL) {` accordingly, or change back as suggested by @kaspar030 - but be consistent

Reply:
Changed back to original style